CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to SDCcc will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
Please add an opening and closing bracket.

Reply:
done